assets/javascripts/discourse/templates/components/babble-composer.hbs

```diff
@@ -9,4 +9,16 @@
   <button {{action 'selectEmoji'}} class="btn no-text emoji" title="Emoji :smile:" aria-label="Emoji :smile:">
     <i class="fa fa-smile-o"></i>
   </button>
+  <button {{action 'upload'}} class="btn no-text upload" title="Upload" aria-label="Upload">
```

Comment:
Note to future self; I think I want to configure whether we should allow file uploads in chat or not; default yes.

Same for emojis probably.